internal/backends/winit/renderer/femtovg/images.rs

```diff
@@ -104,13 +104,7 @@ impl Texture {
                     } else {
                         image_flags
                     };
-                    canvas
-                        .borrow_mut()
-                        .create_image(
-                            &html_image.dom_element,
-                            femtovg::ImageFlags::PREMULTIPLIED | image_flags,
```

Comment:
The `femtovg::ImageFlags::PREMULTIPLIED` is still applied for SVGs a few lines above - in case $reviewer is looking for it :)